### ISSUE TITLE
Dragging items in Files View Stopped Working

### ISSUE THREAD

**Issue by storyjesse** [typora/typora-issues#4528]
- [x] Searched existing issues to avoid creating duplicates.

- [x] Confirmed that it can be reproduced in built-in themes without customized css. *(I haven't customized any css)*

   If it only exists in 3rd party themes or css, you can still report it, but please attach the theme target or the css file. We may not "fix" it, if it is caused by 3rd party themes or css styles that we cannot support.

- [X] Searched http://support.typora.io/

----

**Describe the bug**
After Upgrade to Version: 0.11.0-1 Drag and Drop is not working in the Files Sidebar. I can't move a file to a different folder nor drag in a hyperlink to that file into the document I'm editing.

**To Reproduce**
1. Go to 'Files Sidebar'
2. Click on 'any file shown'
3. Try to drag the file to a new location or into the document
4. Nothing happens.

**Expected behavior**
It used to be possible to do this. Confirmed that rolling back to version 0.10.11 removes the issue. i.e. This issue does not exist in [Version: 0.10.11](https://typora.io/linux/typora_0.10.11_amd64.deb)

**Desktop (please complete the following information):**
 - OS: 
```
System:    Host: Limbo Kernel: 5.4.0-77-generic x86_64 bits: 64 Desktop: Cinnamon 4.4.8 Distro: Linux Mint 19.3 Tricia 
Machine:   Type: Desktop System: MSI product: MS-7823 v: 1.0 serial: <root required> 
           Mobo: MSI model: CSM-H87M-G43 (MS-7823) v: 1.0 serial: <root required> BIOS: American Megatrends v: 1.6 
           date: 02/22/2014 
CPU:       Quad Core: Intel Core i7-4790 type: MT MCP speed: 900 MHz min/max: 800/4000 MHz 
Graphics:  Device-1: NVIDIA GM107GL [Quadro K2200] driver: nvidia v: 390.143 
           Display: x11 server: X.Org 1.19.6 driver: nvidia unloaded: fbdev,modesetting,nouveau,vesa 
           resolution: 1920x1080~60Hz 
           OpenGL: renderer: N/A v: N/A 
```
**Typora Version**
0.11.0

**Comment by abnerlee** [comment]
@storyjesse I'm not able to reproduce this, so possible to attach a screencast about this issue?

**Comment by storyjesse** [comment]
Hi @abnerlee I've just tried to record a video of my desktop showing you the error. But something is going wrong with my recording. Kazam seems to work for short videos and I've used it for a long 40min video before, but I've tried twice now to record about 5 to 10 min of video but nothing is hanppening when I click finish. :-(
Will try again tomorrow.

**Comment by storyjesse** [comment]
OK I've got Kazam working now.
Here's a link to a video showing the bug.
https://youtu.be/mbPPVz2WMaw

**Comment by Loris1123** [comment]
I can confirm this issue. I'm currently on 0.11.2(beta) and drag and drop is not working.

I'm on Manjaro Linux.
Please feel free to ask for the information you need to trace down this issue.

**Comment by GamalielMasters** [comment]
I can also reproduce this issue running ArchLinux with Typora 0.11.8(beta). An attempt to begin a drag acts like a click selecting the item and any following movement acts as if no button is pressed.

**Comment by stompalot** [comment]
Same here using Ubuntu 20.04 and Typora 0.11.8(beta).

Works as expected using Windows 10 same beta so could be a Linux thing.

**Comment by rand0musername** [comment]
Same bug here with 1.0.3 on Ubuntu 20.04

**Comment by pconrad-fb** [comment]
I'm having the same issue.

**Comment by gmcyber** [comment]
Yep, same bug Pop!_OS 21.10

**Comment by irimi1** [comment]
I just encountered it in 1.3.6 (latest release for Linux) on Arch Linux. Feels like it just broke recently for me although I’m not sure.
My package manager is currently very busy, but once it’s done I’m gonna try to find the version in which it stopped working.

**Comment by blurayne** [comment]
Bug? I'd just say: `var i = File.isLinux ? "Not supported" : "hello world"`

RLY?!_

**Comment by irimi1** [comment]
With the changes to the sidebar in version 1.4.3 drag&drop started to work for me again, at least on Windows. I would assume it does so for Linux and Mac as well.

**Comment by irimi1** [comment]
Now that it’s packaged for my distro, I can say that the bug still persists for me with version 1.4.4.

**Comment by ms-private** [comment]
In version 1.4.7 on Linux Mint, drag&drop in the file tree is still impossible (as it was in each release since at least 1.0).
If this is intended, please explain why.
If it is not intended, please fix it.
Fix exists [Issue 4906](https://github.com/typora/typora-issues/issues/4906#issuecomment-992471402) but I don't want to hack the released files each time a I install a new release.

**Comment by blurayne** [comment]
Let me repeat: var i = File.isLinux ? "Not supported" : "we don't care or test for other systems?"

So when do you folks thin about JUST a little bit more Linux support?!

I suggest every Linux user o try put marktext or obsidian as in the moment best alternatives.

**Comment by abnerlee** [comment]
fixed in 1.7.5

**Comment by storyjesse** [comment]
YAY! I confirm that this is fixed in version 1.7.5 on Linux (x64)

I'll happily pay for a licence now just as I said I would.

**Comment by manu-cyber** [comment]
Yup, works for me as well. Thank you! :)
